jax-maven-plugin/src/main/java/com/github/davidmoten/jaxws/HasSourcesOption.java

```diff
@@ -0,0 +1,9 @@
+package com.github.davidmoten.jaxws;
+
+import java.util.List;
+
+interface HasSourcesOption {
```

Comment:
This interface is unnecessary because schemagen is the only goal that has input source. Just do a check if `this` in `BaseMojo.java` is an instanceof `SchemagenMojo`.

Reply:
Done